app/assets/javascripts/application.js

```diff
@@ -179,7 +179,7 @@ $(document).ready(function() {
   }
 
   $('#modal').click(function() {
```

Comment:
This seems a little too broad - I think we should change this to a callback on the [Bootstrap event](http://getbootstrap.com/javascript/#modals-events) for that specific modal so that we only issue the POST request for the quick user modal.